### PR TITLE
Testing baremetal with New hardware

### PR DESCRIPTION
**Pull request by AmarnatReddy**
Testing bare metal with New hardware
Add conf with new hardware allocated 

Logs : https://jenkins.ceph.redhat.com/view/RHCS%20QE/job/rhceph-upi-pipeline-executor/34/console 
Signed-off-by: Amarnath K <amk@amk.remote.csb>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/quincy/e2e/poc/test_cluster_deployment.yaml [hunk 1]
```diff
@@ -245,12 +246,12 @@ tests:
       abort-on-fail: false
       config:
         haproxy_clients:
-          - cali019
+          - argo018
```
Comment (sunilkumarn417) [inline suites/quincy/e2e/poc/test_cluster_deployment.yaml:249]: can we use node labels here ? by using `get_node_by_id`.

File: suites/quincy/e2e/poc/test_cluster_deployment.yaml [hunk 2]
```diff
@@ -245,12 +246,12 @@ tests:
...
         rgw_endpoints:
-          - cali015:80
-          - cali016:80
-          - cali017:80
-          - cali018:80
+          - argo027:80
```
Comment (sunilkumarn417) [inline suites/quincy/e2e/poc/test_cluster_deployment.yaml:251]: same as above

Comment (AmarnatReddy) [reply suites/quincy/e2e/poc/test_cluster_deployment.yaml]: updated and executed 
Logs : https://jenkins.ceph.redhat.com/view/RHCS%20QE/job/rhceph-upi-pipeline-executor/36/console

**Review by hkadam134 — DISMISSED** [review]
LGTM

**Comment by hkadam134** [comment]
These machines will be permanently allocated for Baremenatl CI?

**Pull request closed without merging**